catalog/data_volume.go

```diff
@@ -0,0 +1,27 @@
+package catalog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+)
+
+func DataSourceVolume() common.Resource {
+	return common.WorkspaceData(func(ctx context.Context, data *struct {
+		Id     string              `json:"id,omitempty" tf:"computed"`
+		Name   string              `json:"name"`
+		Volume *catalog.VolumeInfo `json:"volume_info,omitempty" tf:"computed"`
+	}, w *databricks.WorkspaceClient) error {
+		fmt.Println(data)
```

Comment:
We need to remove this line

Reply:
removed in 714d0c2432d14fdfcf765f9d5204db09936f29c1